a10_octavia/db/repositories.py

```diff
@@ -350,6 +351,16 @@ def get_lb_count_by_flavor(self, session, project_ids, flavor_id):
                 or_(self.model_class.provisioning_status == consts.PENDING_DELETE,
                     self.model_class.provisioning_status == consts.ACTIVE)).count()
 
+    def get_lb_exists_flag(self, session, project_id, subnet_id):
```

Comment:
can we change the API name?
get_lb_exists_flag() is confusing, because we return False when lb exists.

Reply:
Done
Renamed it to `check_lb_with_distinct_subnet_and_project()`